SR2022/2021-10-03-sr2022-competition-announcement.md

```diff
@@ -0,0 +1,30 @@
+---
+to: Potential team leaders
+subject: Announcing Student Robotics 2022
+---
+
+Hello there!
+
+We're pleased to announce sign-ups are now open for Student Robotics 2022!
+
+## Competition
+
+This year is the long-awaited return of the physical robotics competition! We're really excited to be going back to what we do best: An intense, exciting, and adrenaline-fuelled competition.
+
+In order to allow the core team of volunteers to focus on delivering a high quality programme while requiring far fewer resources than before, this year's competition will be slightly smaller than previous years. We expect to be able to increase the size of the competition again in the future.
+
+## Kickstart
+
+Kickstart will be streamed live on **13th November 2021**. Here we'll announce the game for this year, go into further details on how the competition year will work, and of course answer any questions you have.
+
+If you already have a kit from a previous Student Robotics competition, please keep hold of it. We'll be in touch soon about getting it updated to the latest version of our kit software. If you don't have a kit, we'll also contact you soon to coordinate shipping a kit out to you.
+
+Visit [our website](TODO) for more information.
+
+## Sign up
+
+If you would like to compete in Student Robotics 2022, please fill in the form at the link below with the required information. We'll announce further details about the kickstart event and competition information to the contact details you provide. The closing date will be *6th October*.
```

Comment:
This closing date is alarmingly close at this point...

Reply:
I don't think this was changed from the previous iteration. Have fixed in 12054ae